phoenix-client-parent/phoenix-client-embedded/pom.xml

```diff
@@ -81,10 +81,5 @@
       <artifactId>phoenix-hbase-compat-${hbase.compat.version}</artifactId>
       <optional>false</optional>
     </dependency>
-    <dependency>
```

Comment:
I thought that we are already not adding sqlline here.

Just noting that we change sqlline-embedded too.